README.md

```diff
@@ -1,5 +1,13 @@
 # MetaMask/action-create-release-pr
 
+This action creates a new release branch and PR. It also makes a commit that includes any release preparation that could be easily autoamted, which at the moment includes the version bump and a partial changelog update (using `@metamask/auto-changelog`).
```

Comment:
Typo: `autoamted`.  I'd also suggest removing `at the moment`

Reply:
Thanks! Done in bdffe96